servo/runner.py

```diff
@@ -341,7 +341,8 @@ def run(self, *, poll: bool = True, interactive: bool = False) -> None:
                 s, lambda s=s: asyncio.create_task(self._shutdown(loop, signal=s))
             )
 
-        loop.set_exception_handler(self._handle_exception)
+        # loop.set_exception_handler(self._handle_exception)
+        loop.set_exception_handler(None)
```

Comment:
Please revert this or implement it properly with toggle-able config set to use `self._handle_exception` by default

Reply:
Debugging code, nice catch thx